ceph.spec.in

```diff
@@ -457,6 +447,34 @@ module derived from Calamari) and expose CLI hooks.  ceph-mgr gathers
 the cluster maps, the daemon metadata, and performance counters, and
 exposes all these to the python modules.
 
+%package mgr-dashboard
+Summary:        ceph-mgr dashboard plugin
```

Comment:
May I suggest to replace this as follows?
```suggestion
Summary:        Ceph Manager Dashboard
```

Reply:
updated and repushed. shall i update `debian/control` as well? currently, it reads

> Description: dashboard plugin for ceph-mgr

Reply:
Yes please, for consistency purposes

Reply:
@LenzGr after a second thought, how should a typical user interpret "Ceph Manager Dashboard"? at the first glimpse, it sounds like a dashboard for ceph-mgr. but actually it's a GUI for interacting with the Ceph cluster. so i feel that "dashboard plugin for ceph-mgr" is a better short summary than "Ceph Manager Dashboard".

what do you think?

Reply:
@tchaikov While technically speaking, the dashboard is a mgr module/plugin, that is really just an implementation detail that isn't of interest to the typical dashboard user. From the user's perspective the dashboard is the "Ceph front-end" and in the documentation it is referred to as the "Ceph Manager Dashboard". This is the name used in various blog posts, videos, presentations, etc.

Reply:
I agree... either "Ceph Dashboard" or "dashboard plugin for ceph-mgr" seem more accurate.

Reply:
see, e.g., http://docs.ceph.com/docs/master/mgr/dashboard/

Reply:
I'm fine with changing the name to just "Ceph Dashboard" (as the reference to "Manager" is indeed confusing/irrelevant).

Reply:
@smithfarm the description is for a user who knows nothing about Ceph . see https://www.debian.org/doc/debian-policy/ch-binary.html#s-descriptions .

> This is the name used in various blog posts, videos, presentations, etc.

in different context, we might want to use different notation. when a user reads http://docs.ceph.com/docs/master/mgr/dashboard/, he/she has much more context to understand and explore the in and out of this software. 

in this context, we don't have the luxury to attract a user. the summary or short description is the only thing that user will see in a package management software's UI, before he/she decides to push Enter and to take a closer look. i think the most important thing are:

- what it does.
- how it could interact with the rest of the system.

i just don't want to take the risk to lose a new user who is not interested in a dashboard merely for mgr. and meh ... what's mgr anyway?

Reply:
it seems like a very odd thing to worry about; I can't see the value in removing information that could be useful.  The fact that it's a plugin for ceph-mgr is important to know, as admins must sometimes audit and enable/disable features by using the mgr module commands.  It doesn't lose status by being called a plugin.

Reply:
yeah, agreed. in the latest version of `debian/control`:
```
Description: dashboard plugin for ceph-mgr
```
and in `ceph.spec.in`
```
Summary:        Ceph Dashboard
```
to accommodate to the comments from Lenz and Sage.